apps/src/applab/applab.js

```diff
@@ -782,7 +784,10 @@ Applab.init = function (config) {
         });
       }
     });
-
+  // Only send PROJECT_ACTIVITY event if they're on a project level.
+  if (!!config.level.isProjectLevel) {
+    analyticsReporter.sendEvent(EVENTS.PROJECT_ACTIVITY, {}, PLATFORMS.BOTH);
```

Comment:
I can't recall the difference between `PROJECT_ACTIVITY` and `LEVEL_ACTIVITY`. Could you add to the PR description a sentence or two describing them?

Reply:
Updated the description with the event names.